themes/stanford_basic/package.json

```diff
@@ -16,7 +16,7 @@
     "autoprefixer": "^10.4.16",
     "babel-loader": "^9.1.2",
     "css-loader": "^6.8.1",
-    "decanter": "^6.2.14",
+    "decanter": "^6.3.2",
```

Comment:
FWIW, `^6.2.14` still captures 6.3.2... But should there be an updated yarn.lock file with this?

Reply:
I added the yarn.lock file. But, why would `^6.2.14` pick up the newer version? I'm missing something fundamental here.

Reply:
It's because of the caret `^`. It allows for versions up to a certain version. `^6.2 14` will download all versions up to 7.0.0. So it will download even 6.99.99 if a version existed 

You can see this using this near semver calculator: https://semver.npmjs.com/